setup.cfg

```diff
@@ -4,7 +4,7 @@ exclude =
 ignore = W503
 
 [isort]
-known_third_party=aiosqlite,ciso8601,quart,sanic
+known_third_party=aiosqlite,ciso8601,quart,sanic,starlette
```

Comment:
I think add `uvicorn` to fix py3.5

Reply:
Yeap, it helped. Forgot about adding this. Thanks!